fabtests/common/shared.c

```diff
@@ -3372,6 +3397,18 @@ uint64_t ft_init_cq_data(struct fi_info *info)
 	}
 }
 
+int ft_check_cq_data()
+{
+	if (rx_cq_entry_has_data) {
+		if (rx_cq_data != ft_init_cq_data(fi)) {
```

Comment:
A call like 'init' should only be called once.  Calling this every time there's a check against the data looks wrong.